### PR TITLE
Combine TAF lists

### PR DESCRIPTION
**Pull request by IvanaEscobar**
NOTE: Transfer PR from stale repo: [PR #4](https://github.com/christophernhill/gmao_mitgcm_couplng/pull/4)

`F90list` is no longer necessary as a separate file. Added contents of `F90list` to `list` which determines which files are passed to TAF for generation of the adjoint model.

---
@mjlosch suggested the following updates: 
> I suggest to also remove `nobcs` in the `SIZE*.h` and break a long STORE directive line in two in `exf_geos_bulkf.F` to make it work "out of the box" with [MITgcm/MITgcm#709](https://github.com/MITgcm/MITgcm/pull/709).

---
I was able to compile and run using `checkpoint68z` of MITgcm. 

Martin was also able to compile:
> That's great, now this works with PR [MITgcm/MITgcm#709](https://github.com/MITgcm/MITgcm/pull/709) for me without any modifications (just add a `FORTRAN_VERS_TAF='F90'` or `ALWAYS_USE_F90=1` to `genmake_local`)
---
Transferred PR per request of @menemenlis

### CODE REVIEW

**Review by jm-c — APPROVED** [review]
LGTM

**Comment by jm-c** [comment]
@IvanaEscobar I am going to merge this PR into master, if it's OK with you. I think it will help for the documentation of https://github.com/MITgcm/MITgcm/pull/709.

File: geos_bulkformulae/global_ocean.cs32x15/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+### Get  MITgcm
+```
+git clone git@github.com:MITgcm/MITgcm.git
+cd MITgcm
+```
+
+### Get code for bulk formulae
+```
+git clone git@github.com:geos-mitgcm-ecco/gmao_mitgcm_coupling.git
```
Comment (IvanaEscobar) [inline geos_bulkformulae/global_ocean.cs32x15/README.md:12]: Updated repo address

**Comment by IvanaEscobar** [comment]
Added `README.md` with recipe instructions on AD model compilation compatible with [PR #709 of MITgcm](https://github.com/MITgcm/MITgcm/pull/709).

NOTE: change in repo address at the top of the `README.md` file

**Pull request merged**